backend/peephole/peephole_utils.ml

```diff
@@ -0,0 +1,50 @@
+(* CR-soon gtulba-lecu for gtulba-lecu: drop the -4 flag *)
```

Comment:
Yes, maybe drop `-4` here and
disable the warning only for
function `is_bitwise_op`.